packages/eslint-plugin/src/rules/ban-instanceof-html-element/index.js

```diff
@@ -2,18 +2,13 @@
 const { AST_NODE_TYPES } = require('@typescript-eslint/utils');
 const createRule = require('../../utils/createRule');
 
-/**
```

Comment:
because of the change [here](https://github.com/microsoft/fluentui/pull/31988/files#r1677194354) it's stopped resolving this import and started to throw ts error. Another way how to fix it, export the type from `@fluentui/react-utilities`, but it's maked as internal, plus after exporting it will make this package kind dependent on that

Reply:
so this was always a code-smell, good to see that 3rd party tools are preventing to do these things haha.

@bsunderhus a todo for you in order to come up with a solution how to keep this in sync in the future. ty

@mainframev 
- please also remove the `export` from `HTMLElementConstructorName` including `@internal` annotation within react-utilities.
- please add more docs to `const constructorNames ` about the type origin etc

ty

Reply:
Not sure we even need to keep that type in sync TBH 🤔, we could just support all possible `HTMLElement` types on the eslint rule and keep it narrower on `@fluentui/react-utilities` I guess. I'll follow up with a PR and link here 🫡

Reply:
Here goes: https://github.com/microsoft/fluentui/pull/32072